scripts/build/version_info.js

```diff
@@ -0,0 +1,80 @@
+// GATHER VERSION INFO FROM COMPONENTS
+const path = require('path');
+const fs = require('fs');
+const {cwd} = process;
+
+// helpers
+const clean = name => name.replace(/^@axa\-ch\//, 'axa-');
```

Comment:
@markus-walther maybe on Windows OS is `\` so please add that in the regex as well

Reply:
pretty sure is not relevant just double check to be safe

Reply:
I checked, and https://docs.npmjs.com/misc/scope mandates a forward slash in package.json names, which is what `clean(...)` operates over.

So I conclude it's fine to leave it as it is.